.github/workflows/wheels.yml

```diff
@@ -199,3 +199,110 @@ jobs:
           source ci/upload_wheels.sh
           set_upload_vars
           upload_wheels
+
+  build_free_threaded_wheels:
+    needs: build_sdist
+    name: Build wheel for ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+    if: >-
+      (github.event_name == 'schedule') ||
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'pull_request' &&
+      contains(github.event.pull_request.labels.*.name, 'Build'))
+    runs-on: ${{ matrix.buildplat[0] }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # GitHub Actions doesn't support pairing matrix values together, let's improvise
+        # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
+        buildplat:
+        - [ubuntu-22.04, manylinux_x86_64]
+        - [ubuntu-22.04, musllinux_x86_64]
+        - [macos-13, macosx_x86_64]
+        - [macos-14, macosx_arm64]
+        python: ["cp313t"]

```

Comment:
We can't just add this to the python matrix above?

Reply:
We could, but the two jobs have enough differences that I though a new job would be warranted.

Reply:
I agree it would be nice to try and consolidate - what are the differences? Probably overlooking, but on a cursory glance I see the Windows handling - anything else?

Reply:
Done. Thanks both for the review!

Reply:
> Probably overlooking, but on a cursory glance I see the Windows handling - anything else?

Another difference, probably a bit more important was that free-threaded wheels were not being uploaded to the staging environment on push. But that's probably ok as well.